local_planner/src/nodes/common.cpp

```diff
@@ -75,29 +61,29 @@ double azimuthIndexToAngle(int z, double res) {
   return z * res + res / 2 - 180;
 }
 
-int azimuthAnglefromCartesian(const geometry_msgs::Point& position,
-                              const geometry_msgs::Point& origin) {
-  return azimuthAnglefromCartesian(position.x, position.y, origin);
+int azimuthAnglefromCartesian(const Eigen::Vector3f& position,
+                              const Eigen::Vector3f& origin) {
+  return azimuthAnglefromCartesian(position.x(), position.y(), origin);
 }
 
-int azimuthAnglefromCartesian(double x, double y,
-                              const geometry_msgs::Point& pos) {
-  return floor(atan2(x - pos.x, y - pos.y) * 180.0 / M_PI);  //(-180. +180]
+int azimuthAnglefromCartesian(double x, double y, const Eigen::Vector3f& pos) {
```

Comment:
Again, a note that this is in degrees would be nice. Also I'm still confused why this needs to be an integer

Reply:
Yep, I want `float`s for everything here. `double`s are overkill (we aren't doing space probe navigation or mm-precision world-scale photogrammetry), and `int`s lead to strange behavior with rounding.